src/components/calcite-tooltip/resources.ts

```diff
@@ -0,0 +1,11 @@
+import Popper from "popper.js";
+
+export const CSS = {
+  container: "container",
+  containerOpen: "container--open",
+  containerFont: "font-size-3",
```

Comment:
I think we can remove this, seems out of place for a utility css class and its already specified in the container.